modules/tinymce/src/core/main/ts/api/OptionTypes.ts

```diff
@@ -300,6 +300,7 @@ export interface EditorOptions extends NormalizedEditorOptions {
   font_size_style_values: string;
   forced_root_block: string;
   forced_root_block_attrs: Record<string, string>;
+  force_hex_color: boolean;
```

Comment:
We should also add `force_hex_color?: boolean;` to the BaseOptions interface around line 45 of this file

Reply:
Added it to BaseEditorOptions as well